content/en/news/releases/1.18.x/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: 1.18.x Releases
+description: Announcements for the 1.18 release and its associated patch releases.
+weight: 12
```

Comment:
The weight should be decremented, so will be 11, so this goes to the top of the generated list.